src/applications/edu-benefits/1990e/containers/IntroductionPage.jsx

```diff
@@ -3,117 +3,164 @@ import { focusElement } from 'platform/utilities/ui';
 import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import WizardContainer from '../../wizard/containers/WizardContainer';
+import {
+  WIZARD_STATUS_NOT_STARTED,
+  WIZARD_STATUS_COMPLETE,
+} from 'applications/static-pages/wizard';
+import { connect } from 'react-redux';
+import { showEduBenefits1990EWizard } from '../../selectors/educationWizard';
 
 export class IntroductionPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      wizardStatus:
+        sessionStorage.getItem('wizardStatus') || WIZARD_STATUS_NOT_STARTED,
+    };
+    this.setWizardStatus = this.setWizardStatus.bind(this);
+  }
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  setWizardStatus(value) {
```

Comment:
For future reference, if you use `setWizardStatus = value => {` here, you won't need to add that `.bind(this);` in the constructor.

Reply:
Thanks! Just refactored.